src/Solocraft.cpp

```diff
@@ -184,9 +138,9 @@ class solocraft_player_instance_handler : public PlayerScript {
             // InstanceScript *instanceScript = instanceMap->GetInstanceScript();
 
             // Announce to player
-            std::ostringstream ss;
-            ss << "|cffFF0000[SoloCraft] |cffFF8000" << player->GetName() << " entered %s - # of Players: %d - Difficulty Offset: %d.";
-            ChatHandler(player->GetSession()).PSendSysMessage(ss.str().c_str(), map->GetMapName(), numInGroup, difficulty);
+           // std::ostringstream ss;
+           // ss << "|cffFF0000[SoloCraft] |cffFF8000" << player->GetName() << " entered %s - # of Players: %d - Difficulty Offset: %d.";
+           // ChatHandler(player->GetSession()).PSendSysMessage(ss.str().c_str(), map->GetMapName(), numInGroup, difficulty);
 
```

Comment:
why removed? why not announce but only to group or to the player only (which was the case already no?)?

Reply:
My bad that was a personal setting of mine I’ll adjust that